src/server/services/tips/predicates.ts

```diff
@@ -32,6 +32,13 @@ export const hasValidId: TipsPredicateFN = (
   });
 };
 
+// To use an ID for voting it needs an expiration date with a maximum of five years ago.
+export const hasValidIdForVoting: TipsPredicateFN = (appState) => {
+  const today = new Date();
+  const fiveYearsAgo = new Date(today.setFullYear(today.getFullYear() - 5));
+  return hasValidId(appState, fiveYearsAgo);
+};

```

Comment:
```suggestion
export const hasValidIdForVoting: TipsPredicateFN = (appState, today) => {
  const fiveYearsAgo = new Date(today.setFullYear(today.getFullYear() - 5));
  return hasValidId(appState, fiveYearsAgo);
};

```

Reply:
2e argument van een predicate is de datum. Zie https://github.com/Amsterdam/mijn-amsterdam-frontend/blob/main/src/server/services/tips/collect-tips.ts#L26